src/ripple/app/main/Application.cpp

```diff
@@ -1976,6 +1976,17 @@ bool ApplicationImp::loadOldLedger (
                 }
             }
         }
+        using namespace std::chrono_literals;
+        static constexpr char ledgerWarnTime[] {"Jan 01, 2018 00:00:00 UTC"};
```

Comment:
Consider inlining `Jan 01, 2018 00:00:00 UTC` into the log message versus declaring a local.

Reply:
My personal preference is to have the `WarnTime` string be immediately adjacent to the `WarnTimePoint`.  I think that increases the likelihood that, if the are changed, they will be maintained together.

Of course the best option is to generate the string from the `time_point` as suggested here: https://github.com/ripple/rippled/pull/3212#discussion_r364531737. We'll have to see if that can be done.